frontend/src/components/ui/AppDocumentUpload.vue

```diff
@@ -108,7 +153,11 @@ export default {
   },
   methods: {
     addFile() {
-      this.documents.push({ id: uuid.v1(), entityName: this.entityName, isValidFile: true })
+      if (this.documentType) {
+        this.documents.push({ id: uuid.v1(), entityName: this.entityName, isValidFile: true, documentType: this.documentType })
```

Comment:
Is there any harm in just pushing this.documentType all the time? Worst case it's undefined and an empty String gets sent to Dynamics.

Reply:
Updated, confirmed on harm.